crates/node/src/scheduler/mod.rs

```diff
@@ -482,6 +482,13 @@ impl TaskManager for Scheduler {
                 running_task.task_started.elapsed().as_secs()
             );
 
+            if let Err(err) = self.database.mark_tx_executed(&running_task.task.tx).await {
+                tracing::error!(
+                    "failed to update transaction.executed => true - tx.hash: {}",
+                    &running_task.task.tx
+                );
+            }
+
```

Comment:
I created a separate PR for this to get it incorporated ASAP: https://github.com/gevulotnetwork/gevulot/pull/114

Reply:
Ok I see I'll rebase when merged